modules/ismp/pallets/token-gateway/Cargo.toml

```diff
@@ -33,12 +34,15 @@ alloy-sol-types = { workspace = true }
 
 token-gateway-primitives = { workspace = true }
 pallet-hyperbridge = { workspace = true }
+pallet-balances = { workspace = true }
+pallet-assets = { workspace = true}
```

Comment:
Actually either put this in dev-dependecies or remove them entirely and use the `fungibles::*` traits as david suggested